cli/azd/pkg/tools/docker/docker.go

```diff
@@ -263,6 +263,10 @@ func (d *Cli) CheckInstalled(ctx context.Context) error {
 	if !supported {
 		return &tools.ErrSemver{ToolName: d.Name(), VersionInfo: d.versionInfo()}
 	}
+	// Check if docker daemon is running
+	if _, err := tools.ExecuteCommand(ctx, d.commandRunner, "docker", "ps"); err != nil {
+		return fmt.Errorf("the Docker daemon is not running, please start the Docker service: %w", err)
```

Comment:
From line L256, it looks like we use `d.Name()` instead of using `Docker` label directly.

This is an easy way to keep the label defined in just one place.  
Please define a local variable before line 256 like `toolName := d.Name()` and then use `toolName` in this error.

Reply:
@vhvb1989 , I have updated by your comments, please re-review.